### ISSUE TITLE
SelectableListBehavior - switch between tabindex 0/-1 based on focused state

### ISSUE THREAD

**Issue by jurokapsiar** [microsoft/fluent-ui-react#1843]
### Problem description
When SelectableListBehavior is used, List should have:
- tabIndex 0 if the focus is outside of the list
- tabindex -1 if the focus is in the list

When the list is focused, focus should move to the first item.

See https://a11y-guidelines.orange.com/web_EN/exemples/listbox/index.html
